package.json

```diff
@@ -19,7 +20,7 @@
   },
   "preconstruct": {
     "packages": [
-      "packages/*"
+      "packages/[!gatsby]"
```

Comment:
TIL about this syntax, thanks!

Reply:
I got it wrong, it should be:

```
      "packages/[!gatsby]*"
```

Semi-related, if I wanted a list of packages preconstruct is taking charge of, how would I do that? Some form of `info` command.

(It's very low priority)

Reply:
That doesn't currently exist, I'm thinking of making a package to do that rather than a command, any objections to that?

Reply:
None at all other than curiousity as to why making it a separate package?